src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -289,6 +268,10 @@ protected override bool ReleaseHandle()
         private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
+
+        internal SafeSslHandle(IntPtr validSslPointer, bool ownsHandle) : base(validSslPointer, ownsHandle)
```

Comment:
I know that I've done this in other places. The implementation doesn't really refer to it as being invalid, just merely the initial value to use. The signature does say invalid, but I always interpreted that as "if you're making a default ctor, make this be something which will make IsInvalid be true"